src/phoenix/server/api/mutations/dataset_mutations.py

```diff
@@ -154,7 +159,111 @@ async def add_spans_to_dataset(
                     for dataset_example_rowid, span in zip(dataset_example_rowids, spans)
                 ],
             )
-        return AddSpansToDatasetPayload(
+        return DatasetMutationPayload(
+            dataset=Dataset(
+                id_attr=dataset.id,
+                name=dataset.name,
+                description=dataset.description,
+                created_at=dataset.created_at,
+                updated_at=dataset.updated_at,
+                metadata=dataset.metadata_,
+            )
+        )
+
+    @strawberry.mutation
+    async def add_examples_to_dataset(
+        self, info: Info[Context, None], input: AddExamplesToDatasetInput
+    ) -> DatasetMutationPayload:
+        dataset_id = input.dataset_id
+        # Extract the span rowids from the input examples if they exit
+        span_ids = [
+            cast(GlobalID, example.span_id)
+            for example in input.examples
+            if example.span_id is not UNSET
+        ]
+        span_rowids = {
+            from_global_id_with_expected_type(global_id=span_id, expected_type_name=Span.__name__)
+            for span_id in set(span_ids)
+        }
+        dataset_version_description = (
+            input.dataset_version_description
+            if isinstance(input.dataset_version_description, str)
+            else None
+        )
```

Comment:
```suggestion
        dataset_version_description = (
            input.dataset_version_description
            if input.dataset_version_description is not UNSET
            else None
        )
```

Suggestion: This pattern doesn't require us to know the type of the field.